tests/EdgeToEdge/Routes/CreditCardPaymentNotificationRouteTest.php

```diff
@@ -61,13 +62,17 @@ public function testGivenValidRequest_applicationIndicatesSuccess() {
 			);
 
 			$client->request(
-				'POST',
+				'GET',
 				'/handle-creditcard-payment-notification',
 				$this->newRequest()
 			);
 
 			$this->assertSame( 200, $client->getResponse()->getStatusCode() );
-			$this->assertSame( 'successful', $client->getResponse()->getContent() );
+			$this->assertContains( 'status=ok', $client->getResponse()->getContent() );
+			$this->assertContains(
+				'url=http://my.donation.app/show-donation-confirmation?donationId=1&accessToken=my_secret_access_token',
+				$client->getResponse()->getContent()
```

Comment:
Not sure, but is it nice to bind to the argument order in the URL here? Would doing multiple asserts for the ket/value pairs not be better?